edit-post/index.js

```diff
@@ -20,7 +20,6 @@ import './hooks';
 import Layout from './components/layout';
 import store from './store';
 import { initializeMetaBoxState } from './store/actions';
-
```

Comment:
Change looks unrelated.

The newline was intended to separate the export statement from the imports.